Library/Formula/sops.rb

```diff
@@ -0,0 +1,120 @@
+class Sops < Formula
+  desc "Secrets OPerationS (sops) is an editor of encrypted files"
```

Comment:
You don't need to include the name in the description.  Just something like "Encrypted file editor" is better.